### PR TITLE
docs: Fix typo in cache-updates doc about optimistic updates

### PR DESCRIPTION
**Pull request by romain-trotard**
Hello :)

## Summary

There is a little typo in an optimistic update in a code example

## Set of changes

Just update param name.

Thanks

### CODE REVIEW

File: docs/graphcache/cache-updates.md [hunk 1]
```diff
@@ -475,7 +475,7 @@ that imitates the result that the API is assumed to send back:
 ```js
 const cache = cacheExchange({
   optimistic: {
-    favoriteTodo(args, cache, info) {
+    favoriteTodo(variables, cache, info) {
       return {
         __typename: 'Todo',
         id: variables.id,
```
Comment (kitten) [inline docs/graphcache/cache-updates.md:481]: Hiya :wave: While this should indeed read as `args`, this line would also need to be updated 😅❤️

Comment (romain-trotard) [reply docs/graphcache/cache-updates.md:481]: Hi Phil. Ah yes with the documentation above `args` is better. 
I chose `variables` like the template just below. Do you want me to change this one too?

I mean change this:
``` 
const cache = cacheExchange({
  optimistic: {
    favoriteTodo(variables, cache, info) {
      return {
        __typename: 'Todo',
        id: variables.id,
        favorite(args, cache, info) {
          return true;
        },
      },
    },
  },
});
```

to:

``` 
const cache = cacheExchange({
  optimistic: {
    favoriteTodo(args, cache, info) {
      return {
        __typename: 'Todo',
        id: args.id,
        favorite(args2, cache, info) { // Here I changed to `args2` but can let `args` as you want :)
          return true;
        },
      },
    },
  },
});
```

Comment (kitten) [reply docs/graphcache/cache-updates.md:481]: I'd just call it `_args` and accept that shadowing the name even is fine ✌️ 
Ultimately, it's just a code sampel that conveys meaning, and `args` functionally over `variables` is more important than the exact name 🙌

Comment (romain-trotard) [reply docs/graphcache/cache-updates.md]: I made the changes ;)
Hope it's fine :D

**Review by kitten — APPROVED** [review]
Cheers! ❤️‍🔥

**Pull request merged**